src/main/java/br/com/caelum/vraptor/panettone/VRaptorCompilationListener.java

```diff
@@ -19,7 +14,8 @@
 	public Variable[] getExtraInjections() {
 		return new Variable[] {
 				new Variable("javax.servlet.http.HttpServletResponse", "res"),
-				new Variable("br.com.caelum.vraptor.Result", "result")};
+				new Variable("br.com.caelum.vraptor.Result", "result"),
+				new Variable("br.com.caelum.vraptor.panettone.api.PrintEscaper", "printEscaper")};
```

Comment:
This is the right place in the current implementation, we might change it a
little bit if rolling back the interface implenetation next week, but don't
worry until then

On Tue, Apr 28, 2015 at 10:16 AM Caio Oliveira <notifications@github.com>
wrote:

> In
> src/main/java/br/com/caelum/vraptor/panettone/VRaptorCompilationListener.java
> <https://github.com/caelum/vraptor-panettone/pull/72#discussion_r29207648>
> :
>
> > @@ -19,7 +14,8 @@
> >  	public Variable[] getExtraInjections() {
> >  		return new Variable[] {
> >  				new Variable("javax.servlet.http.HttpServletResponse", "res"),
> > -				new Variable("br.com.caelum.vraptor.Result", "result")};
> > +				new Variable("br.com.caelum.vraptor.Result", "result"),
> > +				new Variable("br.com.caelum.vraptor.panettone.api.PrintEscaper", "printEscaper")};
>
> @guilhermesilveira <https://github.com/guilhermesilveira> I have a doubt.
> The declaration of this variable is in the correct place ? Or should be in
> PanettoneWalker?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/caelum/vraptor-panettone/pull/72/files#r29207648>.
>